### ISSUE TITLE
offset filter?

### ISSUE THREAD

**Issue by cameronjroe** [leizongmin/tinyliquid#41]
Does the offset filter work?

**Comment by cameronjroe** [comment]
?

**Comment by leizongmin** [comment]
Did you mean `offset` option in `for` tag, such as `{% for item in list offset:3 %}`?

**Comment by cameronjroe** [comment]
Yeah, that's a filter in the Liquid Templating Spec.

**Comment by leizongmin** [comment]
Yes, it works well in `tinyliquid`. But that's something you need to be noticed: for example, in `offset:3` that's no spaces between `offset` and `3`, and the `3` must be a number, not a variable (such as `offset:n`)

**Comment by cameronroe** [comment]
I'm confused. Are you saying that `offset:3` is using the `3` differently than `offset: 3`?

**Comment by cameronroe** [comment]
That's not really ideal if it's the case. Ideally all non-whitespace should be handled the same way.
